src/Controls/src/Core/PublicAPI/net-ios/PublicAPI.Shipped.txt

```diff
@@ -7849,9 +7849,7 @@ static Microsoft.Maui.Controls.LayoutDirectionExtensions.ToFlowDirection(this Mi
 static Microsoft.Maui.Controls.LayoutOptions.operator !=(Microsoft.Maui.Controls.LayoutOptions left, Microsoft.Maui.Controls.LayoutOptions right) -> bool
 static Microsoft.Maui.Controls.LayoutOptions.operator ==(Microsoft.Maui.Controls.LayoutOptions left, Microsoft.Maui.Controls.LayoutOptions right) -> bool
 static Microsoft.Maui.Controls.Platform.Extensions.ToPlatformModalPresentationStyle(this Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific.UIModalPresentationStyle style) -> UIKit.UIModalPresentationStyle
-static Microsoft.Maui.Controls.Platform.FormattedStringExtensions.ToNSAttributedString(this Microsoft.Maui.Controls.FormattedString! formattedString, Microsoft.Maui.IFontManager! fontManager, double defaultLineHeight = 0, Microsoft.Maui.TextAlignment defaultHorizontalAlignment = Microsoft.Maui.TextAlignment.Start, Microsoft.Maui.Font? defaultFont = null, Microsoft.Maui.Graphics.Color? defaultColor = null, Microsoft.Maui.TextTransform defaultTextTransform = Microsoft.Maui.TextTransform.Default) -> Foundation.NSAttributedString!
```

Comment:
You can t delete this from the shipped, you need to add the removed to Unshipped

Reply:
Fixed